src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs

```diff
@@ -100,6 +101,14 @@ public Model(string modelUrl, string resourceDirectoryUrl = "") : this(Interop.M
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             this.PositionUsesPivotPoint = true;
+            ResourcesLoaded += (e, s) =>
```

Comment:
Usually, we don't recommend to use instance callback in platform level.

Could you please make it as seperated method, and remove signal at `Dispose` time?

Reply:
Thank you I fixed it.